src/strings/en-us.json

```diff
@@ -1624,7 +1624,7 @@
     "HeaderSelectFallbackFontPath": "Select Fallback Font Folder Path",
     "HeaderSelectFallbackFontPathHelp": "Browse or enter the path of the fallback font folder to use for rendering ASS/SSA subtitles.",
     "LabelFallbackFontPath": "Fallback font folder path",
-    "LabelFallbackFontPathHelp": "Specify a path containing fallback fonts for rendering ASS/SSA subtitles. The maximum allowed total font size is 20 MB. Lightweight and web-friendly font formats such as woff2 are recommended.",
+    "LabelFallbackFontPathHelp": "These fonts are used by some clients to render subtitles. Please refer to the documentation for more info.",
```

Comment:
```suggestion
    "LabelFallbackFontPathHelp": "These fonts are used by some clients to render subtitles when transcoding. Please refer to the documentation for more information.",
```

Reply:
Your suggested change is inaccurate. These fonts aren't used if the server is burning in the subtitles. They are only currently used by the web client to render subtitles client side.

Reply:
:exploding_head: well that makes no sense for a setting in the encoding settings... no wonder people find it confusing

Reply:
I agree that this setting and its description make absolutely no sense, but this is how it is currently and should be documented.